src/components/common/MetaTags/index.tsx

```diff
@@ -10,6 +10,7 @@ const titleText = 'Safe{Wallet}'
 const MetaTags = ({ prefetchUrl }: { prefetchUrl: string }) => (
   <>
     <meta name="description" content={descriptionText} />
+    <meta name="robots" content="noindex" />
```

Comment:
Looks good but it should be there only on dev/staging.

```suggestion
    {!IS_PRODUCTION && <meta name="robots" content="noindex" />}
```

Reply:
Got it!